sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/windowExpressions.scala

```diff
@@ -365,14 +360,6 @@ abstract class OffsetWindowFunction
 
   override def children: Seq[Expression] = Seq(input, offset, default)
 
-  /*
```

Comment:
Can we keep the original comment and just add a new line of comment that says `foldable` is properly set to `false` from `Unevaluable`?

Reply:
+1

Reply:
Added it back